site/static/images/getting-started/tutorials/figma-basics/placeholder

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is the purpose of this file? Should it be an image file?

Reply:
On the browser I couldn't figure out how to create a folder for the images without creating a file first :D So it is not needed